RELEASE.md

```diff
@@ -1,78 +1,164 @@
-1. [NGINX](https://github.com/kubernetes/ingress-nginx/tree/main/images/nginx)
+## 1. [NGINX Base Image](https://github.com/kubernetes/ingress-nginx/tree/main/images/nginx)
 
-* Open pull request
+A. Open pull request
   
-If you are updating any component in [build.sh](images/nginx/rootfs/build.sh) please also update the SHA256 checksum of that component as 
-  well, the cloud build will fail with an exit 10 if not.  
+  If you are updating any component in [build.sh](images/nginx/rootfs/build.sh) please also update the SHA256 checksum of that component as well, the cloud build will fail with an exit 10 if not.  
 Example [NGINX_VERSION](images/nginx/rootfs/build.sh#L21),
 [SHA256](images/nginx/rootfs/build.sh#L124) 
-* Merge
-* Wait for [cloud build](https://console.cloud.google.com/cloud-build/builds?project=k8s-staging-ingress-nginx)
 
-1a. Promote images:
+B. Merge
 
-Open pull request to promote staging image:
-[add sha - version](https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml#L1)
+C. Wait for [cloud build](https://console.cloud.google.com/cloud-build/builds?project=k8s-staging-ingress-nginx)
+
+D. Promote Images:
+
+  Open pull request to promote staging image:
+  [add sha - version](https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml#L1)
 
 The sha is available in output from [cloud build](https://console.cloud.google.com/cloud-build/builds?project=k8s-staging-ingress-nginx)
```

Comment:
I think we should add an additional note for people who don't have access to cloud build (like me). The sha and version can also be found here: https://console.cloud.google.com/gcr/images/k8s-staging-ingress-nginx/GLOBAL

Reply:
Can we write that in a follow up PR  James knows more about the cloudbuild